src/base-command.ts

```diff
@@ -63,6 +64,16 @@ export default abstract class BaseCommand extends Command {
     }
   }
 
+  async finally(_: Error | undefined): Promise<any> {
+    if (this.app.config.environment !== ENVIRONMENT.TEST && !_) {
+      const { filtered_sentry_args, filtered_sentry_flags } = await this._getNonSensitiveSentryMetadata();
```

Comment:
This code looks duplicated, please move it to a new function that is just called from both places